test/compile_test/interpolators_vector_barycentric_rational_incl_test.cpp

```diff
@@ -0,0 +1,9 @@
+//  Copyright Matt Borland 2021.
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// A sanity check that this file
+// #includes all the files that it needs to.
+//
+#include <boost/math/interpolators/vector_barycentric_rational.hpp>
```

Comment:
Instantiation?

Reply:
@NAThompson `vector_barycentric_rational` is the only one of the interpolators not in the `interpolators` namespace. Do you want me to move it or leave it? Currently it is in just `boost::math`.

Reply:
Yes, that was a screw up on my part.